package.json

```diff
@@ -12,6 +12,7 @@
   "author": "FIT3170-2021 Team 2",
   "license": "ISC",
   "dependencies": {
-    "concurrently": "^6.1.0"
+    "concurrently": "^6.1.0",
+    "lint": "^0.7.0"
```

Comment:
@rsun0013 you can remove `lint` and rollback the associated `package-lock.json` changes as these are not needed. You can also rollback the  `client/package-lock.json` changes as we're not updating any dependencies in this PR.

Reply:
@rsun0013 https://docs.npmjs.com/cli/v7/commands/npm-uninstall